com.sap.cloud.lm.sl.cf.persistence/src/main/java/com/sap/cloud/lm/sl/cf/persistence/services/FileSystemFileService.java

```diff
@@ -106,6 +108,18 @@ private Path getFilePath(FileEntry entry) throws IOException {
     public int deleteAll(String space, String namespace) throws FileStorageException {
         throw new UnsupportedOperationException();
     }
+    
+    @Override
+    public void deleteSpaceLeftovers(String space) throws FileStorageException {
+        File spaceDirectory = getSpaceDirectory(space).toFile();
```

Comment:
I believe the getSpaceDirectory brings some duplication here.
See:

>    private Path getSpaceDirectory(String space) {
>         return Paths.get(storagePath, space);
>     }

And this:

> private Path getFilesPerSpaceDirectory(String space) {
>     return Paths.get(storagePath, space, DEFAULT_FILES_STORAGE_PATH);
> }

What do you think?

Reply:
Done